core/src/mindustry/game/MapObjectives.java

```diff
@@ -612,6 +616,17 @@ public void drawMinimap(MinimapRenderer minimap){}
         public void added(){}
         /** Remove any UI elements, if necessary. */
         public void removed(){}
+        /** Control marker with world processor code*/
+        public void control(LMarkerControl type, float... values){
```

Comment:
Don't use varargs, they allocate arrays! There should be N parameters, where N is the maximum amount of parameters that may ever be used.